lib/dbservice_web/controllers/student_controller.ex

```diff
@@ -135,26 +135,6 @@ defmodule DbserviceWeb.StudentController do
     end
   end
 
-  swagger_path :register do
-    post("/api/student/register")
-
-    parameters do
-      body(:body, Schema.ref(:StudentRegistration), "Student to create along with user",
-        required: true
-      )
-    end
-
-    response(201, "Created", Schema.ref(:StudentWithUser))
-  end
```

Comment:
Why swagger path for student registration is removed?

Reply:
We already have a swagger path for create, no.

Reply:
Ideally this should now be:
`
swagger_path :register do
    post("/api/student")

    parameters do
      body(:body, Schema.ref(:StudentRegistration), "Student to create along with user",
        required: true
      )
    end

    response(201, "Created", Schema.ref(:StudentWithUser))
  end
`

The Schema references should change.